frontend/src/app/signUp/Organization/utils.tsx

```diff
@@ -125,10 +99,7 @@ export const organizationSchema: yup.SchemaOf<OrganizationCreateRequest> = yup
       .mixed()
       .oneOf(Object.keys(OrganizationTypeEnum), "Organization type is required")
       .required(),
-    state: yup
-      .mixed()
-      .oneOf(liveJurisdictions, getStateErrorMessage)
-      .required(),
+    state: yup.string().required(),
```

Comment:
non-blocking nit and not 100% related to your actual ticket: can we specify the error message here? right now it gives me `state is a required field` which is not completely the same as what we get for the other fields

Reply:
What error would you expect for this?  Current error message is consistent with other required fields (like org type).

Reply:
Oh I think I see now `s required` vs `is a required field`?

Reply:
`State is required`

it looks like all the other required fields have a specific error message set instead of using what I assume is auto-generated by yup